scripts/test-code.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
 echo MATRIX_SCALA=$MATRIX_SCALA
-sbt -DscalaJSStage=full ++$MATRIX_SCALA test publishLocal || exit 1
+sbt -DscalaJSStage=full \
+    -J-XX:MinRAMPercentage=90.0 \
```

Comment:
Why would it be required?

Reply:
It's because Scala Native requires more RAM to build the binary with the tests.
`-XX:MinRAMPercentage=90.0` and `-XX:MaxRAMPercentage=90.0` allow us to use the RAM available in the machine (especially in the Github Action executor) without making assumptions about its memory size.